js/webrtc.js

```diff
@@ -576,12 +708,12 @@ var spreedMappingTable = [];
 				userIndicator.className = 'nameIndicator';
 				if (peer) {
 					if (peer.nick) {
-						userIndicator.textContent = peer.nick;
+						userIndicator.textContent = t('spreed', "{participantName}'s screen", {participantName: peer.nick});
```

Comment:
This does not work for all languages, we need to look into this at some point.
Even in english: `foobas's screen` is incorrect...